drivers/video/sunxi/sunxi_display.c

```diff
@@ -46,6 +47,25 @@
 #define PWM_OFF 0
 #endif
 
+#define COLOR_RED 0xFFFF0000
+#define COLOR_GREEN 0xFF00FF00
+#define COLOR_BLUE 0xFF0000FF
+#define COLOR_BLACK 0xFF000000
+#define COLOR_WHITE 0xFFFFFFFF
+#define COLOR_TRANSPARENT 0x00000000
+#define SAVE_X_OFFSET
```

Comment:
Why it's empty? I don't see any usage in code. It may be for letter spaccing because now its bad TBH

Reply:
It is used to set start offset on X axis, but I will 
 remove it

Reply:
yeah didn't notice #ifdef for it, but text_x_start is either way set to 0